Assets/MRTK/Core/Providers/InputSimulation/ManualCameraControl.cs

```diff
@@ -31,6 +31,12 @@ private static float InputCurve(float x)
             return Mathf.Sign(x) * (1.0f - Mathf.Cos(0.5f * Mathf.PI * Mathf.Clamp(x, -1.0f, 1.0f)));
         }
 
+
+        public void SetInitialTransform(Transform transform)
```

Comment:
Please add a `<summary>` here to describe when / how this should be used.

Also, since it's only currently used by other input simulation classes, could it be `internal`? Or do you expect this method to need to be called manually by a dev as well?

Reply:
Added the `<summary>`.  Given the `UpdateTransform` method is public (and making that internal would be a breaking change), I'm inclined to keep it the same - if you're calling `UpdateTransform` you should really call `SetInitialTransform` too.